airflow/opt/providers/etna/etna/tests/conftest.py

```diff
@@ -271,7 +271,6 @@ def no_op(*args, **kwargs):
             "sync_appbuilder_roles",
             "init_jinja_globals",
             "init_xframe_protection",
-            "init_permanent_session",
```

Comment:
For some reason, when I ran tests locally using the `app` fixture, I was getting an exception that this method `init_permanent_session` didn't exist. Perhaps it was deprecated and removed recently?

Reply:
Huh.  Possible.  I think I might have _started_ using the app fixture but backed out from using it for some reason.  I'm trying to remember the context.  

We did upgrade the airflow version at one point, so if this was added before the upgrade but not used after, that's plausible.

Reply:
Looks like it was taken out in February 2022: https://github.com/apache/airflow/commit/da9d0863c7ff121c111a455708163b026943bdf1#diff-048ecd9f5ec6ccdc46b6cd7740cb9dcce877562817d3e2f997d2df39d2dcefec